searchlib/src/vespa/searchlib/attribute/changevector.h

```diff
@@ -25,32 +25,32 @@ struct ChangeBase {
         DIV,
         CLEARDOC
     };
-    enum {UNSET_ENUM = 0xffffffffu};
+    enum {UNSET_ENTRY_REF = 0xffffffffu};
```

Comment:
Note that this value is within the range of valid entry refs thus we might (very rarely) consider a set entry ref to not being set. Consider using 0 instead.

Reply:
Fixed in https://github.com/vespa-engine/vespa/pull/20633.